.evergreen/atlas_data_lake/drivers-test-local.yaml

```diff
@@ -0,0 +1,66 @@
+environment: development
+
+hosting:
+  frontend:
+    bindAddrs: ["localhost:27017"]
+    cursor:
+      metadata:
+        memory: true
+  showInternalErrors: true
+
+execution:
+  mqlrunPath: ${MONGOHOUSE_MQLRUN}
+
+tenants:
+  config:
+    inline:
+      hostnames:
+        - localhost
+      security:
+        roles:
+          # Based on the atlasAdmin custom role defined in NDSDBUser.java in the
+          # 10gen/mms repo.
+          - role: atlasAdmin
```

Comment:
Do we actually need all of this for the driver ADL testing? I would think `readAnyDatabase` would be sufficient, with no additional privileges.

My concern here is just paring this down to the bare minimum required to test and eliminating anything that was casually copied from another source -- in the interest of making this easier to maintain/understand down the line.

Reply:
This role always exists and is sent to ADL from Atlas (when Atlas is used as the control plane). In this case, it is mimicking what Atlas gives us.

But no, it probably doesn't need to exist if the `mhuser` role below is assigned a different role, like `readAnyDatabase`.

Reply:
Thanks for clarifying. I'd opt for just using `readAnyDatabase` if that's all we need to run the tests in https://github.com/mongodb/specifications/pull/821 then.